### ISSUE TITLE
Can't ssh into the VM on Windows 

### ISSUE THREAD

**Issue by jeffmaury** [crc-org/crc#3701]
### General information

  * OS: Windows 10 Pro
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes
  * Running CRC on: Laptop
  * 
## CRC version
```bash
CRC version: 2.20.0+f3a947
OpenShift version: 4.13.0
Podman version: 4.4.4
```
  
## CRC status
```bash
DEBU CRC version: 2.20.0+f3a947
DEBU OpenShift version: 4.13.0
DEBU Podman version: 4.4.4
DEBU Running 'crc status'
CRC VM:          Running
MicroShift:      Running (v4.13.0)
RAM Usage:       1.17GB of 3.902GB
Disk Usage:      4.836GB of 16.1GB (Inside the CRC VM)
Cache Usage:     27.44GB
Cache Directory: C:\Users\demo\.crc\cache
```

## CRC config
```bash
- preset                                : microshift
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```

### Steps to reproduce

  1. Start CRC
  2. Ssh into the VM

### Expected

SSH enters the VM

### Actual

The private key is accessible to current user (demo) and another one (None ?). Removing the last makes SSH working again
```
C:\Users\demo>ssh -i ~/.crc/machines/crc/id_ecdsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null  core@127.0.0.1 -p 2222
Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@         WARNING: UNPROTECTED PRIVATE KEY FILE!          @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
Permissions for 'C:\\Users\\demo/.crc/machines/crc/id_ecdsa' are too open.
It is required that your private key files are NOT accessible by others.
This private key will be ignored.
Load key "C:\\Users\\demo/.crc/machines/crc/id_ecdsa": bad permissions
core@127.0.0.1's password:

```

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by gbraad** [comment]
> Load key "C:\\Users\\demo/.crc/machines/crc/id_ecdsa": bad permissions

This is a local issue. That happens when you are using the 'wrong' ssh client.
CRC itself is not affected by this.

WONTFIX

**Comment by praveenkumar** [comment]
https://superuser.com/questions/1296024/windows-ssh-permissions-for-private-key-are-too-open
